grobid-service/config/config.yaml

```diff
@@ -30,11 +30,4 @@ logging:
   appenders:
     - type: console
       threshold: ALL
-      timeZone: UTC
```

Comment:
This change would impact the log file in general, I would make the modification on the fly when creating the image

Reply:
The intention was not log to a file at all as that will eventually fill up the allocated storage space for the container. If the preference is to have a log file by default, I can drop this change. Making the change at container build time in a robust manner would require adding a YAML parser to the builder (e.g. https://github.com/biojppm/rapidyaml).  Is that OK?

Reply:
@lfoppiano Any thoughts?

Reply:
Hello ! 

In the next GROBID version, this service config file will disappear and will be replaced by a unique yaml config file for Grobid (replacing the grobid.properties too), see PR #687 

Then we can modify and mount the config file at launch of the docker container:
> docker run --rm --gpus all --init -v /home/lopez/grobid/grobid-home/config/grobid.yaml:/opt/grobid/grobid-home/config/grobid.yaml:ro  grobid/grobid:0.6.2-SNAPSHOT

and this way the log will be defined for the service at the container run time, which is more flexible.  

So I would say the default log is less important (personally I don't like the logs on the console, which I find unusable).

Reply:
Those new config changes are great. In our production setup we modify both config.yaml & grobid.properties before copying them in to the container, so having just one file to manage will be an improvement.

I'll drop my change to config.yaml here since they are somewhat orthogonal to the size issue.

Reply:
Just for references, my idea was to keep the original config and adapt it based on the needs in docker. For example removing that lines in the config file via bash scripting when building the docker image.  

Anyway, indeed with the config file passed to the container the problem is solved at the source ;-)